### DISCUSSION TITLE
Feature Request: Suggest Comment Edits

### DISCUSSION THREAD

**Discussion by chharvey** [community/community#4208] [Other Feature Feedback, Questions, & Ideas]
*This is a copy of my post on the [GitHub Support Community](https://github.community/t/feature-request-suggest-comment-edits/153076) site.*

I would like to request a feature that enables users to suggest edits to comments in Issues, Pull Requests, and Discussions. After a user suggests an edit, the author of the comment would be given a chance to accept, reject, or modify the edit.

Suggested edits could look a bit like the [suggest code change](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) feature in Pull Requests, where the author could see the diff of what’s been changed. If multiple suggestions are made before the author addresses them, they could be merged or shown side-by-side. There could even be settings that control who has permissions to suggest edits, i.e. only those who have commenting permissions.

I think this feature would be useful for updating outdated information, correcting typos, improving formatting, etc. without having to add a new comment to the thread and notify everyone subscribed. Suggested edits would only notify the author of the comment, and reduce the amount of clutter in a thread.

This isn’t a novel idea. A similar concept appears on StackOverflow where users with enough reputation may edit a question/answer, and then it must be approved by a more trusted user.

**Comment by jimmywarting** [comment]
I like the idea of having something more like StackOverflow where you earn privileges, Suggest edits. and even something like https://stackoverflow.com/review/suggested-edits where the community can help
